.github/workflows/test-jest.yml

```diff
@@ -20,3 +20,5 @@ jobs:
         working-directory: ./launcher
       - run: npm run test
         working-directory: ./launcher
+        env:
+          HCLOUD_TOKEN: ${{ secrets.HCLOUD_TOKEN }}
```

Comment:
can we get a separate github action for JEST integration tests? The new JEST integration test job shouldn't run on PRs, just on merges on branch `main` like the molecule tests. This is because the `secrets.HCLOUD_TOKEN` could be exposed by an attacker just doing something like `log.debug(HCLOUD_TOKEN)`.